docs/walkthrough/loading-models.ipynb

```diff
@@ -81,43 +81,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
```

Comment:
could you keep this output? I purposely included it in the docs for loading models.

Reply:
yeah can do. I was just running the npm targets and expecting that was the prep we wanted. Maybe there's a way to exclude some of them... I'll check